### ISSUE TITLE
Issues with latest WMI installer

### ISSUE THREAD

**Issue by thiagoftsm** [netdata/msi-installer#30]
#### Issue 1

When  I tried to install and claim a node today on `Windows Server 2022` I received this error:

![Screenshot_20221201_171850](https://user-images.githubusercontent.com/49162938/205121807-d9d4363d-cce2-49d3-94cc-744bda1d9765.png)

And I could not finish installation. 
After this double click installer and everything worked as expected.

#### Issue 2

During installation time on the same server, it cannot use WSL2 and it uses WSL1:

![Screenshot_20221201_172444](https://user-images.githubusercontent.com/49162938/205122067-eb29e8ea-e5fb-4c72-9ec7-fd9072c0fed4.png)

I am not sure if this is an error with installer and our image, or an issue on server. :thinking: 

#### Issue 3

We are not using latest version of [windows exporter](https://github.com/prometheus-community/windows_exporter/releases/tag/v0.20.0). Is there a specific reason for this? I am using the latest to develop and I do not identify an error that can postpone  our use.

#### Issue 4
@cakrit we are always installing Windows Exporter with default options instead to detect possible servers installed and enable options. Should we improve this a little bit?

**Comment by dfpr** [comment]
> #### Issue 1
> When I tried to install and claim a node today on `Windows Server 2022` I received this error:
> 
> ![Screenshot_20221201_171850](https://user-images.githubusercontent.com/49162938/205121807-d9d4363d-cce2-49d3-94cc-744bda1d9765.png)
> 
> And I could not finish installation. After this double click installer and everything worked as expected.

MSIEXEC won't open `.\netdata.msi`, the correct command is `msiexec.exe /i netdata.msi` without the `.\` prefix.
Double click will only work for the built-in administrator account that has UAC disabled.

> #### Issue 2
> During installation time on the same server, it cannot use WSL2 and it uses WSL1:
> 
> ![Screenshot_20221201_172444](https://user-images.githubusercontent.com/49162938/205122067-eb29e8ea-e5fb-4c72-9ec7-fd9072c0fed4.png)
> 
> I am not sure if this is an error with installer and our image, or an issue on server. 🤔

Server doesn't have Virtualization or Nested virtualization available so it has to use WSL1.
Task manager | Performance | CPU will say "Virtualization: Enabled" only when available.

> #### Issue 3
> We are not using latest version of [windows exporter](https://github.com/prometheus-community/windows_exporter/releases/tag/v0.20.0). Is there a specific reason for this? I am using the latest to develop and I do not identify an error that can postpone our use.

I guess that version was released after I started working on the installer, anyway, I have updated it to v0.20.

**Comment by thiagoftsm** [comment]
Thank you for your explanation, I will keep it open until we update `Windows Exporter`.

**Comment by dfpr** [comment]
Latest release includes windows exporter v0.20

**Comment by thiagoftsm** [comment]
Hello @dfpr ,

Thank you to update the `Windows Exporter`. Now about to claim a node, we still have an issue on `Windows Server`, please, take a look in what happens when I use Power Shell to install:

![fail](https://user-images.githubusercontent.com/49162938/205179225-ba9b1994-53b3-47af-ac48-b2b7cf9f10e0.png)

On the other hand, when I do not use additional information necessary to claim a node, the installation happens as expected:
![success](https://user-images.githubusercontent.com/49162938/205179348-498d8fd9-05bd-4e19-8f6e-98587b3d36ad.png)

Best regards!

**Comment by thiagoftsm** [comment]
> Latest release should handle rooms with spaces in their names correctly.

Yes, but I used quotes around them.

**Comment by dfpr** [comment]
msiexec received the variable correctly but invocation of netdata-claim.sh didn't, that is fixed

**Comment by thiagoftsm** [comment]
Hello @dfpr ,

Unfortunately the latest release still has the issue:

![Screenshot_20221202_002511](https://user-images.githubusercontent.com/49162938/205187455-be4f1e0c-9d0b-4466-a885-3e41fc02b647.png)

I kept the Room name this time to help you, and I also showed part of the key too.

Best  regards!

**Comment by dfpr** [comment]
Sorry, I thought the issue was in the installation script but it is just a powershell issue: double quotes inside the command need a backtick place before: ```msiexec /i netdata.msi ROOMS=`"My room`"``` or even better don't run the command using powershell but use cmd.

**Comment by thiagoftsm** [comment]
> Sorry, I thought the issue was in the installation script but it is just a powershell issue: double quotes inside the command need a backtick place before: `` msiexec /i netdata.msi ROOMS=`"My room`" `` or even better don't run the command using powershell but use cmd.

Nice, now I could run the command and install `netdata` from Power Shell, but the VM restarted without to ask me nothing. When I ran `msiexec.exe /i netdata.msi` I had the expected behavior asking me to reboot.

**Comment by dfpr** [comment]
Non-interactive restart only happened to me with the AUTORESTART=1 argument, I'll test deeper this weekend.

**Comment by thiagoftsm** [comment]
> Non-interactive restart only happened to me with the AUTORESTART=1 argument, I'll test deeper this weekend.

All right, thank you!  
I never set AUTORESTART=1 while I was installing, probably we have something inside our codes. :thinking:
Please, let me know if you need any additional information.

**Comment by dfpr** [comment]
I could take a look at the installation log at C:\netdata.log

**Comment by thiagoftsm** [comment]
Hello @dfpr ,

Sorry for the delay, [this](https://github.com/netdata/msi-installer/files/10161948/log.log) is the output of the command you gave me. 

For the second time I had server restarting while I was installing. 

About `I'd say don't use powershell but use cmd`, well I think we need to detect the prompt and raise a message box for users stopping the installation and redirecting them for the normal `cmd`.

Please, let me know if you need any other thing.

Best regards!

**Comment by dfpr** [comment]
I'll check deeper how to avoid the issue with powershell and try to show message boxes when the command line is malformed
